src/Evaluator.hs

```diff
@@ -353,7 +352,7 @@ step (Up v e k) =
         Up (ValueMacroAction
```

Comment:
for some reason I can't annotate line 351, but the same `env` 'defined but not used' is there as well.